lib/va_profile/prefill/military_information.rb

```diff
@@ -0,0 +1,372 @@
+        discharge_type
+        guard_reserve_service_history
+        hca_last_service_branch
+        is_va_service_connected
+        last_discharge_date
+        last_entry_date
+        last_service_branch
+        latest_guard_reserve_service_period
+        post_nov111998_combat
+        service_branches
+        service_episodes_by_date
+        service_periods
+        sw_asia_combat
+        tours_of_duty
+        va_compensation_type
+      ].freeze
+
+      HCA_SERVICE_BRANCHES = {
+        'A' => 'army',
+        'C' => 'coast guard',
+        'F' => 'air force',
+        'H' => 'usphs',
+        'M' => 'marine corps',
+        'N' => 'navy',
+        'O' => 'noaa'
+      }.freeze
+
+      DISCHARGE_TYPES = {
+        'A' => 'honorable',
+        'B' => 'general',
+        'D' => 'bad-conduct',
+        'F' => 'dishonorable',
+        'J' => 'honorable',
+        'K' => 'dishonorable'
+      }.freeze
+
+      SOUTHWEST_ASIA = %w[
+        AM
+        AZ
+        BH
+        CY
+        GE
+        IQ
+        IL
+        JO
+        KW
+        LB
+        OM
+        QA
+        SA
+        SY
+        TR
+        AE
+        YE
+      ].freeze
+
+      NOV_1998 = Date.new(1998, 11, 11)
+      GULF_WAR_RANGE = (Date.new(1990, 8, 2)..NOV_1998)
+
+      # Disability ratings counted as lower
+      LOWER_DISABILITY_RATINGS = [10, 20, 30, 40].freeze
+
+      # Disability ratings counted as higher
+      HIGHER_DISABILITY_RATING = 50
+
+      # In https://github.com/department-of-veterans-affairs/va.gov-team/issues/41046
+      # military service branches were modified to use an updated list from
+      # Lighthouse BRD. The list below combines the branches from the former list
+      # that was in the previous vets_json_schema, and the new list, from BRD. In
+      # the future, consider udpating this constant to a dynamic value populated
+      # by a call to Lighthouse BRD.
+      COMBINED_SERVICE_BRANCHES = [
+        'Army Air Corps or Army Air Force',
+        'Air Force Academy',
+        'Air Force',
+        'Air Force Reserve',
+        'Air Force Reserves',
+        'Air National Guard',
+        'Army Reserve',
+        'Army Reserves',
+        'Army',
+        'Army National Guard',
+        'Coast Guard Academy',
+        'Coast Guard',
+        'Coast Guard Reserve',
+        'Coast Guard Reserves',
+        'Marine Corps',
+        'Marine Corps Reserve',
+        'Marine Corps Reserves',
+        'Merchant Marine',
+        'Naval Academy',
+        'Navy',
+        'National Oceanic & Atmospheric Administration',
+        'NOAA',
+        'Navy Reserve',
+        'Navy Reserves',
+        'Other',
+        'Public Health Service',
+        'Space Force',
+        'US Military Academy',
+        "Women's Army Corps"
+      ].freeze
+
+      attr_reader :disability_service, :military_personnel_service
+
+      def initialize(user)
+        @disability_service = VAProfile::Disability::Service.new(user)
```

Comment:
I would consider refactoring `user` out if possible, `Disability::Service` only appears to use `edipi` (so you can just pass that instead)

Reply:
Ah, ok. I think we'd done that because the base class accepts `user` as the argument. But that makes sense, fixing now.